lib/moab/storage_services.rb

```diff
@@ -14,80 +14,79 @@ module Moab
   #   All rights reserved.  See {file:LICENSE.rdoc} for details.
   class StorageServices
     # @return [StorageRepository] an instance of the interface to SDR storage
```

Comment:
I was expecting this to `include Singleton` see: https://ruby-doc.org/stdlib-3.0.0/libdoc/singleton/rdoc/Singleton.html

Reply:
ok, i think i made this change as suggested (in the second commit).

i added a test, but i'm not sure it's a worthwhile test?